app/src/main/java/com/asfoundation/wallet/ui/Erc681ReceiverPresenter.java

```diff
@@ -59,7 +59,7 @@ public void present(Bundle savedInstanceState) {
     return throwable instanceof WalletNotFoundException ? createWallet() : Single.error(throwable);
   }
 
-  public void pause() {
+  void pause() {
```

Comment:
Shouldn't the method present() be as well package protected for the sake of consistency?

Reply:
Agree, but the Ide didn't warn me about that XD